examples/Makefile

```diff
@@ -1,25 +1,95 @@
 .PHONY: opencv object_detection delete minikube
 
+GATK_GERMLINE_FILES	=	gatk/GATK_Germline/data/ref/ref.dict \
+				gatk/GATK_Germline/data/ref/ref.fasta \
+				gatk/GATK_Germline/data/ref/ref.fasta.fai \
+				gatk/GATK_Germline/data/ref/refSDF/done \
+				gatk/GATK_Germline/data/ref/refSDF/format.log \
+				gatk/GATK_Germline/data/ref/refSDF/mainIndex \
+				gatk/GATK_Germline/data/ref/refSDF/namedata0 \
+				gatk/GATK_Germline/data/ref/refSDF/nameIndex0 \
+				gatk/GATK_Germline/data/ref/refSDF/namepointer0 \
+				gatk/GATK_Germline/data/ref/refSDF/progress \
+				gatk/GATK_Germline/data/ref/refSDF/seqdata0 \
+				gatk/GATK_Germline/data/ref/refSDF/seqpointer0 \
+				gatk/GATK_Germline/data/ref/refSDF/sequenceIndex0 \
+				gatk/GATK_Germline/data/ref/refSDF/summary.txt \
+				gatk/GATK_Germline/data/bams/father.bai \
+				gatk/GATK_Germline/data/bams/father.bam \
+				gatk/GATK_Germline/data/bams/mother.bai \
+				gatk/GATK_Germline/data/bams/mother.bam \
+				gatk/GATK_Germline/data/bams/motherICE.bai \
+				gatk/GATK_Germline/data/bams/motherICE.bam \
+				gatk/GATK_Germline/data/bams/motherNEX.bai \
+				gatk/GATK_Germline/data/bams/motherNEX.bam \
+				gatk/GATK_Germline/data/bams/motherRnaseq.bai \
+				gatk/GATK_Germline/data/bams/motherRnaseq.bam \
+				gatk/GATK_Germline/data/bams/motherRnaseqPP.bai \
+				gatk/GATK_Germline/data/bams/motherRnaseqPP.bam \
+				gatk/GATK_Germline/data/bams/son.bai \
+				gatk/GATK_Germline/data/bams/son.bam
+
 opencv:
 	pachctl create repo images
 	pachctl create pipeline -f opencv/edges.json
 	pachctl create pipeline -f opencv/montage.json
 	pachctl put file images@master -i opencv/images.txt
 	pachctl put file images@master -i opencv/images2.txt
 
-object_detection:
+ml/object-detection/frozen_inference_graph.pb:
+	wget -p http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz --directory-prefix /tmp
+	tar -C ml/object-detection --strip-components 1 -xvf /tmp/download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz ssd_mobilenet_v1_coco_11_06_2017/frozen_inference_graph.pb
+
+object_detection: ml/object-detection/frozen_inference_graph.pb
 	pachctl create repo images
 	pachctl create repo training
 	pachctl create pipeline -f ml/object-detection/model.json
 	pachctl create pipeline -f ml/object-detection/detect.json
-	pachctl put file training@master:frozen_inference_graph.pb -f ~/ssd_mobilenet_v1_coco_11_06_2017/frozen_inference_graph.pb
+	pachctl put file training@master:frozen_inference_graph.pb -f ml/object-detection/frozen_inference_graph.pb
 	pachctl put file images@master:dogs.jpg -f ml/object-detection/images/dogs.jpg
 	pachctl put file images@master:kites.jpg -f ml/object-detection/images/kites.jpg
```

Comment:
Broader note:

Is it easy enough to add comments or something like that to a makefile? It'd be nice to have the 3rd object detection images (airplane) or 3rd batch of images for opencv, etc available in a copy/paste instead of copying the previous put from the makefile and editing it

Reply:
done.